front/src/common/redux/forgot-password/forgot.password.action.js

```diff
@@ -0,0 +1,6 @@
+import { FORGOT_PASSWORD } from './types';
+
+export const forgotPasswordAction = (data) => ({
+    type: FORGOT_PASSWORD,
+    data
```

Comment:
Please, use `payload` key instead of `data`

Reply:
done